### DISCUSSION TITLE
Component rendering issues within a Web Embedding

### DISCUSSION THREAD

**Discussion by vrseraphin** [pegasystems/constellation-ui-gallery#52] [General]
For those that may have attempted to take one of the components in this gallery for use as a Custom DX Component and encountering rendering issues when attempting to use that custom component within a Web Embedding, I wanted to raise awareness about this Web Embed troubleshooting topic:  [https://docs.pega.com/bundle/platform/page/platform/user-experience/troubleshooting-web-embeds.html#ariaid-title7](url)

Hope to get all the components in this repo updated to properly wrap any rendered markup within a <Configuration> object to avoid this issue.

**Comment by ricmars** [comment]
This issue has been addressed with this PR https://github.com/pegasystems/constellation-ui-gallery/pull/55
